.github/workflows/simorgh-local-server-tests.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     env:
       CI: true
       LOG_LEVEL: 'error'
+      BFF_PATH: 'https://web-cdn.test.api.bbci.co.uk/fd/simorgh-bff'
```

Comment:
Is there anyway we can avoid adding this here... (open source 'n that) - so that we don't risk someone finding the URL and DDOSing us? Not sure how we have done this elsewhere...?

Reply:
Copied this from 
https://github.com/bbc/simorgh/blob/7d122f4ffc3800f93f52bdd1293f63a2eb34e7f9/.github/workflows/simorgh-integration-tests.yml#L24 so assumed it would be fine as it already exists in repo , but agree its probably not ideal...

Reply:
Ah so it is. That's fine then so, we can leave it here, and hope no one spams us 😟